ar-setup.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+artifact_registry_location=us-central1
+project=$(gcloud config get-value core/project)
+
+usage () {
+    echo "usage: ./ar-setup.sh --app-name <app_name> --gitlab-access-token <token> --app-config-repo <app-config-repo> [--project <gcp_project>] [--artifact-registry-location <location>]"
+	echo "  --app-name (required): the application name."
+	echo "  --app-config-repo (required): the URL to the application project. This is needed so that we can add the AR repository and the Google service account key as project variables of the application project."
+	echo "  --gitlab-access-token (required): the GitLab personal access token. This is needed to access the GitLab APIs."
```

Comment:
Maybe add a link about how to generate a new access token?

Reply:
Also need to specify the scope for that token. https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#creating-a-personal-access-token

For GitLab, we need both `api` and `write_repository` scope

Reply:
This script only needs the `api` scope to add project variables.

Reply:
Done. PTAL.